app/states/texas/validators/texas_expenses.py

```
@@ -187,74 +150,89 @@ class TECExpense(TECSettings, table=True):
-        elif values["payeePersentTypeCd"] == "ENTITY":
-            if not values["payeeNameOrganization"]:
-                raise PydanticCustomError(
-                    'missing_required_value',
-                    "payeeNameOrganization is required for ENTITY payeePersentTypeCd",
-                    {
-                        'column': 'payeeNameOrganization',
-                        'value': values["payeeNameOrganization"]
-                    }
-                )
-        else:
+    def validate_payee(cls, values):
```

Ruff (0.14.7)

Missing return type annotation for classmethod `validate_payee`

(ANN206)